src/java/grl/src/grl/driver/GRL_Driver.java

```diff
@@ -206,15 +206,17 @@ public void run()
 
 		// TODO: Let user set mode (teach/joint control from tablet as a backup!)
 		//this.getApplicationData().getProcessData("DefaultMode").
-		
 
-		
+
+     // Printing dummy message:
+		 getLogger().info("DUMMY: " + _currentKUKAiiwaState.dummy());
```

Comment:
this will probably overload the logger, won't it? should this be in an if(debug) or something like that?